tests/python/pants_test/java/test_nailgun_protocol.py

```diff
@@ -205,6 +205,43 @@ def test_isatty_from_env_mixed(self):
       (False, True, False)
     )
 
+  def _make_mock_stream(self, isatty, fileno):
+    mock_stream = mock.Mock()
+    mock_stream.isatty.return_value = isatty
+    mock_stream.fileno.return_value = fileno
+    return mock_stream
+
+  @mock.patch('os.ttyname', autospec=True, spec_set=True)
+  def test_isatty_to_env_with_mock_tty(self, mock_ttyname):
+    mock_ttyname.return_value = '/dev/ttys000'
```

Comment:
this should probably point to a non-existent path to avoid any inadvertent use of that actual static char device.

Reply:
Oops, should have thought a little harder about that. Fixed in 9d221e6.